inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_convolution_test.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include "myriad_layers_convolution_test.hpp"
 
-INSTANTIATE_TEST_CASE_P(accuracy_chw_dilation, myriadLayerConvolution_smoke,
+INSTANTIATE_TEST_CASE_P(accuracy_chw_dilation, myriadLayerConvolution,
```

Comment:
Why not `..._nightly` instead of `..._smoke`?

Reply:
You remove this test from smoke, but in parameters I see

```
::testing::Values<layoutPreference>(vpu::LayoutPreference::ChannelMajor)
```

Should it have been removed pre-commit?